src/components/NumberAndDate.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { TextField, Grid } from "@mui/material";
+
+export default function NumberAndDate() {
+  return (
+    <>
+      <Grid item xs={5} container spacing={4} direction={"row"}>
+        <Grid item xs={1} />
+        <Grid item xs={10}>
+          <TextField
+            variant="standard"
+            fullWidth
+            id="no"
+            name="no"
+            label="No."
+            type="text"
+            //   value={formik.values.password}
+            //   onChange={formik.handleChange}
+            //   error={formik.touched.password && Boolean(formik.errors.password)}
+            //   helperText={formik.touched.password && formik.errors.password}
+          />
+        </Grid>
+        <Grid item xs={1} />
```

Comment:
Po co sa te puste elementy?

Reply:
zeby sie wszystko ladnie ulozylo :)

Reply:
to nie tędy droga, odezwij się w razie problemów, ale Grid z MUI daje możliwość zrobienia odpowiednich odstepów, czy co tam potrzebujesz. Może trzeba inaczej zaplanować ten układ, ale nie korzystaj z pustych elementów DOM aby poprawić układ, użyj do tego styli css.

Reply:
wszystko posprzatane